src/wallet/rpcwallet.cpp

```diff
@@ -1359,6 +1359,23 @@ static void ListTransactions(interfaces::Chain::Lock& locked_chain, CWallet* con
     }
 }
 
+static const std::string TransactionDescriptionString()
+{
+    return "    \"confirmations\": n,       (numeric) The number of confirmations for the transaction. Negative confirmations indicate the\n"
+           "                                         transaction conflicts with the block chain\n"
```

Comment:
While extracting this, it wouldn't hurt to fixup the nits that wouldn't be worth a commit on their own, like adding the missing period at the end of this sentence and next sentence.

Reply:
e.g. s/chain\n"/chain.\n"/ and s/one\n"/one.\n"/

Reply:
Suggestion: s/Only returns true if transaction is a coinbase one./True if the transaction is a coinbase, otherwise false./

Reply:
It would be nice to align the horizontal spacing of this entire section.